sdk/include/platform/sunburst/platform-i2c.hh

```diff
@@ -233,6 +233,9 @@ struct OpenTitanI2c
 	  FifoControlTransmitReset = 1 << 8,
 	};
 
+	static constexpr uint32_t RXFIFODepth = 8;
+	static constexpr uint32_t FmtFIFODepth = 8;
```

Comment:
Maybe make these `uint8_t` to stop anything more than UINT8_MAX being set here?

Reply:
I prefer to leave it as is, we could also want to configure far deeper FIFOs (they are SRAM based in I2C) and I don't think there's any real value in restricting it. Ultimately if it's not set to an appropriate number it's broken and limiting to UINT8_MAX won't help that.

Reply:
Ok, happy to leave this as is.